docs/cloud-messaging/receive.md

```diff
@@ -200,3 +200,68 @@ Next, the worker must be registered. Within the entry file, **after** the `main.
 
 Next restart your Flutter application. The worker will be registered and any background messages will be handled via this file.
 
+### Handling Interaction
```

Comment:
Hey @kevinthecheung, I've just noticed that [this section](https://firebase.flutter.dev/docs/messaging/notifications#handling-interaction) from the previous documentation is missing on the Firebase docs. I've just reinstated it, let me know if this is acceptable. Thanks!